File: COM1786-CW1-Project-5/COM1786-CW1-Project-5/app/src/main/java/com/example/comp1786cw1project3/local/database/AppDatabase.java
package com.example.comp1786cw1project3.local.database;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.example.comp1786cw1project3.model.Expense;
import com.example.comp1786cw1project3.model.Trip;

@Database(entities = {Trip.class, Expense.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract TripDao tripDao();
    public abstract ExpenseDao expenseDao();
}

File: COM1786-CW2-Project-5/COM1786-CW2-Project-5/app/src/main/java/com/example/comp1786cw2project2/local/database/AppDatabase.kt
package com.example.comp1786cw2project2.local.database

import androidx.room.Database
import androidx.room.RoomDatabase
import com.example.comp1786cw2project2.local.model.Url

@Database(entities = [Url::class], version = 1)
abstract class AppDatabase: RoomDatabase() {
    abstract fun urlDao(): UrlDao
}